src/test/java/com/easypost/ShipmentTest.java

```diff
@@ -77,7 +80,10 @@ public void testAll() throws EasyPostException {
 
         assertTrue(shipmentsList.size() <= Fixture.pageSize());
         assertNotNull(shipments.getHasMore());
-        assertTrue(shipmentsList.stream().allMatch(shipment -> shipment instanceof Shipment));
```

Comment:
Was this accidentally changed?

Reply:
Huh... not sure where how this came about, I'll patch this.